src/model_signing/_utils.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2025 The Sigstore Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Utility functions for CLI tool."""
+
+import pathlib
+from typing import Optional
+
+from cryptography.x509 import ExtensionNotFound
+from cryptography.x509 import RFC822Name
+from cryptography.x509 import SubjectAlternativeName
+from sigstore import models as sigstore_models
+from sigstore.verify.policy import OIDCIssuer
+
+
+def get_oidc_params_from_sigfile(
```

Comment:
This is just for Sigstore signing, right? Should we move it to that file instead? Having a `utils` module is usually an antipattern, trying to avoid it.

Reply:
Will move it.